Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: vault
-version: 0.15.0
```

Comment:
I believe we have some pending changes to release, so no need to bump the version until then.